script/tool/README.md

```diff
@@ -135,6 +135,26 @@ the branch point from `upstream/main`. For more complex use cases where you want
 a different diff point, you can pass a different `--base-branch`, or use
 `--base-sha` to pick the exact diff point.
 
+### Update a dependency
+
+`update-dependency` will update a pub dependency to a new version.
+
+For instance, to updated to version 3.0.0 of `some_package` in every package
+that depends on it:
+
+```sh
+cd <repository root>
+dart run script/tool/bin/flutter_plugin_tools.dart update-dependency \
+  --pub-package=some_package \
+  --version=3.0.0 \
+```
+
+If you don't provied a `--version`, the latest version from pub will be used.
```

Comment:
type: provide

Reply:
Also should probably say something similar to "If a `--version` is not provided"